src/styles/main.scss

```diff
@@ -0,0 +1,87 @@
+* {
+  box-sizing: border-box;
+}
+
+$stopwatchSize: 500px;
+$minuteHandSize: $stopwatchSize/2;
+$minuteHandWidth: 5px;
+$secondHandSize: $stopwatchSize/3;
+$secondHandWidth: 5px;
+$nailSize: $minuteHandWidth * 3;
+
+@mixin angle {
+  &::before {
+    content: "";
+    position: absolute;
+    width: 0;
+    height: 0;
+    border-right: $nailSize/2 solid transparent;
+    border-left: $nailSize/2 solid transparent;
+    border-bottom: $nailSize solid #888;
+    top: calc((0% + #{$nailSize} - 6px) / 2);
+    left: calc((100% - #{$nailSize}) / 2);
+  }
+}
+
+.stopwatch {
+  width: $stopwatchSize;
+  height: $stopwatchSize;
+  border-radius: 50%;
+  border: 1px dotted #000;
+  position: relative;
+  margin-inline: auto;
+  margin-block: calc((100vh - #{$stopwatchSize}) / 2);
```

Comment:
Use 'margin-top: calc((100vh - #{$stopwatchSize}) / 2)' to center the stopwatch vertically.